cfgov/jinja2/v1/_includes/organisms/header.html

```diff
@@ -40,6 +40,17 @@
     </div>
     {% endif %}
 
+    {% if banners %}
+        {% for banner in banners %}
+            <div class="m-global-banner">
+                <div class="wrapper
+                          wrapper__match-content">
+                    {% include_block banner %}
```

Comment:
```suggestion
                    {% include_block banner.content %}
```

This needs to be `banner.content` to include the `content` StreamField as opposed to the Django model instance.

Reply:
Yeah, with the way you suggested reworking the query, that's true.

Reply:
Oh, yes, you're right. I was testing against your branch with that local change applied.